common/middleware/set-services.js

```diff
@@ -0,0 +1,25 @@
+const services = require('../services')
+
+const setServices = (req, res, next) => {
+  const servicesCache = {}
```

Comment:
Might be worth renaming `servicesCache` to `servicesRequestCache` or  something that makes it more explicit that this is a short-term, per-request rather than long-lived cache.

Reply:
Maybe something like servicesRequestContext or something?

Reply:
Unless anyone has other suggestions, `servicesRequestContext` it shall be